seedlet/inc/wpcom-colors-utils.php

```diff
@@ -97,7 +97,7 @@ function rgb_to_hsvl( $rgb ) {
 	$min_rgb           = min( $r, $g, $b );
 	$chroma            = $max_rgb - $min_rgb;
 	$v                 = 100 * $max_rgb;
-	if ( 0 === $chroma ) {
+	if ( 0 == $chroma ) {
```

Comment:
Using a less strict comparison causes the warning to go away, but I had to commit with `--no-verify` to avoid the strict comparison check.

Reply:
Maybe the way round that would be to do `if ( 0 === (int) $chroma )`?